convert2rhel/initialize.py

```diff
@@ -77,4 +77,12 @@ def run():
 
     from convert2rhel import main
 
-    sys.exit(main.main())
+    retval = 0
+    try:
+        with applock.ApplicationLock("convert2rhel"):
+            retval = main.main()
```

Comment:
```suggestion
    try:
        utils.require_root()
        with applock.ApplicationLock("convert2rhel"):
            retval = main.main()
```

We need to check the UID before we initialize the applock, so we get the
```
The tool needs to be run under the root user.

No changes were made to the system.
```
before the PermissionError when trying accessing the `/var/run/lock/convert2rhel{}.pid` as non-root

Reply:
If we go with this, we should remove the call from main, there is no point in having this check twice... But I agree that would be a solution